packages/gatsby-source-contentful/src/source-nodes.js

```diff
@@ -80,9 +80,12 @@ export async function sourceNodes(
     `environment`
   )}`
 
-  const fetchActivity = reporter.activityTimer(`Contentful: Fetch data`, {
-    parentSpan,
-  })
+  const fetchActivity = reporter.activityTimer(
+    `Contentful: Fetch data (${sourceId})`,
```

Comment:
This was done on purpose here: https://github.com/gatsbyjs/gatsby/pull/34860

Reply:
oh ok so I didn't do a bad rebase, I did checkout from release/4.8 or 4.9 so make sense.  This shows we need some kind of ids for these things too